CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ project(
 #
 # <INSERT_CMAKE_INSTALL_PREFIX_HERE>
 #set(CMAKE_INSTALL_PREFIX "/opt/poky/rzn1/sysroots/x86_64-pokysdk-linux/usr")
+#set(CMAKE_INSTALL_PREFIX "/opt/poky/rzn1/sysroots/x86_64-pokysdk-linux/usr")
```

Comment:
get a rid of this line ;) not exactlt necessary to si there....

Reply:
removed

Reply:
Why I do not see tidying up code here ?